test-case/verify-sof-firmware-load.sh

```diff
@@ -22,6 +22,13 @@ func_opt_parse_option "$@"
 
 setup_kernel_check_point
 
+( set +x
+    if [ "$SOF_VERSION_CHECK" = 'none' ]; then
+        printf '$''SOF_VERSION_CHECK=none, skipping verify-sof-firmware-load.sh\n'
+        exit 2
```

Comment:
There is a much more readable `skip_test` function now.

Reply:
Is this what you had in mind?
````
if [ "$SOF_VERSION_CHECK" = 'none' ]; then
        skip_test ''SOF_VERSION_CHECK=none"
fi
````

Reply:
Yes this should work and be equivalent.